spec/support/creation_json.rb

```diff
@@ -0,0 +1,51 @@
+module CreationJson
+  def creation_json
+    {
+      'data' => {
+        'id'   => "Xakl22k",
+        'type' => 'creations',
+        'attributes' => {
+          'name' => 'A boat',
+          'translated_names' => 'abc',
+          'approved' => 'abc',
+          'approval_status' => 'abc',
+          'created_at_age' => 'abc',
+          'created_at_age_per_creator' => 'abc',
+          'reflection_text' => 'abc',
+          'reflection_video_url' => 'abc',
+          'image' => {'links' => {'original' => 'https://api.creatubbles.com/v2/creation/image/boat'}},
+          'image_status' => 'abc',
+          'bubbles_count' => 'abc',
+          'comments_count' => 'abc',
+          'views_count' => 'abc',
+          'last_bubbled_at' => 'abc',
+          'last_commented_at' => 'abc',
+          'last_submitted_at' => 'abc',
+          'short_url' => "https://ctbl.es/Xakl22k",
+          'obj_file_url' => 'abc',
+          'play_iframe_url' => 'abc',
+          'created_at' => 'abc',
+          'updated_at' => 'abc',
+          'tags' => 'abc'
+        },
+        'relationships' => {
+          'user' => {
+            'data' => {
+              'id' => 'xUt6Feou',
+              'type' => 'users'
+            },
+            'links' => {
+              'self' => 'https://api.creatubbles.com/v2/users/xU4lFeou'
```

Comment:
this for example doesn't seem correct. Can you just take a real creation data as a basis for this hash.